### PR TITLE
Add chart_attribute to dashboard.

### PR DESCRIPTION
**Pull request by keep94**

### CODE REVIEW

**Comment by keep94** [comment]
When I run the linter on my local machine it runs clean. When I check the output of the linter in github it says no issues found. I don't know why it keeps failing.

File: wavefront/common.go [hunk 1]
```diff
@@ -15,6 +17,19 @@ func suppressSpaces(k, old, new string, d *schema.ResourceData) bool {
 	return strings.TrimSpace(old) == strings.TrimSpace(new)
 }
 
+func suppressJSONFormatting(k, old, new string, d *schema.ResourceData) bool {
```
Comment (ajackson) [inline wavefront/common.go:20]: I'm a bit confused by the name of this function. I believe it's just testing if the two json blobs can parse and are equal. Maybe something like `validAndEqual` or something?

**Review by ajackson — COMMENTED** [review]
On comment on a function name but the rest LGTM.

File: wavefront/common.go (hunk 1 above)
Comment (keep94) [reply wavefront/common.go:20]: Correct. The idea behind this function is that it is comparing two JSON documents to see that they are semantically equal. That is they express the same thing even though they may have different tabs and whitespace in them.

Comment (keep94) [reply wavefront/common.go:20]: Named changed to isJSONForFieldTheSame.

**Review by ajackson — APPROVED** [review]
LGTM

**Pull request merged**